### ISSUE TITLE
Pulmonary hypertensive crisis

### ISSUE THREAD

**Issue by johannaest** [obophenotype/human-phenotype-ontology#6170]
**Preferred term label:**
Pulmonary hypertensive crisis

**Synonyms**
Pulmonary arterial hypertension crisis

**Definition (free text, please give PubMed ID)**
Definition: Pulmonary hypertensive crisis involves sudden and potentially lethal increases in PAP and PVR that cause acute rise in right atrial and right ventricular end-diastolic pressure accompanied by low cardiac output.
DOI: 10.1513/AnnalsATS.201602-110CME

**Parent term (use hpo.jax.org/app)**
Hypertensive crisis

**Comment by pnrobinson** [comment]
adding term, but as a child of Elevated pulmonary artery pressure -- the hypertension in "Hypertensive crisis" is systemic, and so it does not fit. Thanks for the suggestion!
